### ISSUE TITLE
Can't operate with display

### ISSUE THREAD

**Issue by zakfromevil** [frap129/klipmi#4]
After installing klimpi and flashing TFT firmware, the display shows start page with thumbnail, temperatures, I can switch pages, but can't touch to change anything exclude LIGHT /Sound toggles.

**Comment by frap129** [comment]
Yes, those are the only hardware control features that have been implemented for the OpenQ1 target

**Comment by zakfromevil** [comment]
Can you implement control temps/load-unload filament, and other stock features? Buttons mapping to gcode commands as I think?

Maybe it can be solving by seeing from X3lcd /Neptune 4 projects

**Comment by zakfromevil** [comment]
Maybe it can be solving by seeing  from X3lcd /Neptune 4 projects

**Comment by frap129** [comment]
At some point, I've been quite busy lately and haven't had the time to work on this

**Comment by straga** [comment]
@zakfromevil 
I do some updates - https://github.com/straga/klipmi

**Comment by sonooby** [comment]
> [@zakfromevil](https://github.com/zakfromevil) I do some updates - https://github.com/straga/klipmi

can you add more stock features? if you need more py scripts can download or contact me [https://disk.yandex.ru/d/oUlHk2l1tRh95w](url)
